src/Http/ApiClient.php

```diff
@@ -0,0 +1,193 @@
+<?php
+namespace Jobsity\PhpTick\Http;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Exception\ServerException;
+use mef\Log\StandardLogger;
+
+/**
+ * Class ApiClient
+ *
+ * @package Jobsity\PhpTick\Http
+ */
+class ApiClient implements ClientInterface
+{
+    const BASE_URL = 'https://www.tickspot.com/';
+    const ENDPOINT_URL = '/api/v2/';
+
+    /**
+     * @var string User's subscription id
+     */
+    private $subscriptionId;
+
+    /**
+     * @var string User's access token
+     */
+    private $accessToken;
+
+    /**
+     * @var string User's company
+     */
+    private $company;
+
+    /**
+     * @var string User's email
+     */
+    private $email;
+
+    /**
+    * @var string API URL
+    */
+    private $apiUrl;
+
+    /**
+     * @var Client Guzzle Client Handler
+     */
+    private $client;
+
+    /**
+     * Constructs ApiClient
+     *
+     * @param string   $subscriptionId   Subscription id of the user.
+     * @param string   $accessToken      Access token of the user.
+     * @param string   $company          User's company.
+     * @param string   $email            User's email.
+     */
+    public function __construct($subscriptionId, $accessToken, $company, $email)
```

Comment:
Change constructor to receive the following parameters:

 ```public function __construct(\GuzzleHttp\ClientInterface $client, \mef\Log\Logger $logger, $subscriptionId, $accessToken, $company, $email)```

Create a factory method:
```
public function getInstance($subscriptionId, $accessToken, $company, $email)
```


Remember to add use statements.